indexing-service/src/test/java/org/apache/druid/indexing/common/actions/SegmentAllocateActionTest.java

```diff
@@ -1122,6 +1188,41 @@ private SegmentIdWithShardSpec allocate(
     );
   }
 
+  private SegmentIdWithShardSpec allocateWithoutLineageCheck(
```

Comment:
Were there no existing tests that performed allocation without lineage check?

Reply:
No, there weren't.